web/src/pages/export.tsx

```diff
@@ -0,0 +1,60 @@
+import { CircularIndicator } from "@/components/CircularIndicator";
+import { Roadmap } from "@/components/dashboard/Roadmap";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { TaskBody } from "@/components/TaskBody";
+import { useRoadmap } from "@/lib/data-hooks/useRoadmap";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { exportComponentsAsPDF } from "@/lib/roadmap/exportRoadmapAsPdf";
+import { useRouter } from "next/router";
+import { ReactElement, useEffect, useState } from "react";
+
+const ExportPage = (): ReactElement => {
+  const { roadmap } = useRoadmap();
+  const [downloadIndicator, setDownloadIndicator] = useState(false);
+  const router = useRouter();
+  const FEATURE_EXPORT_PDF = process.env.FEATURE_EXPORT_PDF === "true";
+
+  useEffect(() => {
+    if (!FEATURE_EXPORT_PDF) router.push(ROUTES.dashboard);
+  }, [router, FEATURE_EXPORT_PDF]);
+
+  return (
+    <>
+      {FEATURE_EXPORT_PDF && roadmap ? (
```

Comment:
[sand] two notes:
- It seems strange that if we have this feature disabled, it will just have a forever loading indicator instead of just a 404 or text on the page that says "this is not enabled" or similar. I feel like loading forever will make it seem broken when it's not.
- In general, we prefer these to be a guard clause before the return statement rather than a ternary in the JSX. It makes it much much easier to read and understand what's being rendered when the body of the component isn't nested inside a ternary.

Reply:
```
useEffect(() => {
    if (!FEATURE_EXPORT_PDF) router.push(ROUTES.dashboard);
  }, [router, FEATURE_EXPORT_PDF]);

if (!FEATURE_EXPORT_PDF || !roadmap) return <CircularIndicator />;
```

I've added the suggested guard clause. I have a question about a situation where a user is on this page in an environment where the feature is disabled. They will see the loading indicator until the use effect hook routes them to the dashboard page. Do you have any comments about this behavior? Should we be routing them to the 404 page instead? @aloverso

Reply:
@FarazA22 looks good, makes sense to me. I missed the routing in the useEffect initially, sorry